doc/dev/release.md

```diff
@@ -298,6 +297,32 @@ brew bump-formula-pr --strict --url=$OPERATORSDKURL --sha256=$OPERATORSUM operat
 
 Note: If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
 
+You've now fully released a new version of the Operator SDK. Good work! Make sure to follow the post-release steps below.
+
+### (Post-release) Updating the operator-sdk-samples repo
+
+Many releases change SDK API's and conventions, which are not reflected in the [operator-sdk-samples repo][sdk-samples-repo]. The samples repo should be updated and versioned after each SDK release with the same tag, ex. `v1.3.0`, so users can refer to the correct operator code for that release.
```

Comment:
Do we only need to update samples on major and minor releases?

```suggestion
Many releases change SDK API's and conventions, which are not reflected in the [operator-sdk-samples repo][sdk-samples-repo]. The samples repo should be updated and versioned after each SDK major/minor release with the same tag, ex. `v1.3.0`, so users can refer to the correct operator code for that release.
```

Reply:
Hopefully yes. Our patch releases haven't broken any API's IIRC (nor should they).